pkg/patterns/declarative/pkg/manifest/objects.go

```diff
@@ -331,8 +332,8 @@ func ParseObjects(ctx context.Context, manifest string) (*Objects, error) {
 		out := &unstructured.Unstructured{}
 		err := decoder.Decode(out)
 		if err != nil {
-			log.WithValues("yaml", yaml).Error(err, "error decoding object")
-			return nil, fmt.Errorf("error decoding object: %v", err)
+			log.WithValues("yaml", yaml).Info("Unable to parse into Unstructured, Storing as blob")
```

Comment:
Can you increase the log verbosity here? This isn't something we should see by default